docs/documentation/stories/global-styles.md

```diff
@@ -6,36 +6,41 @@ The `styles.css` file allows users to add global styles and supports
 If the project is created with the `--style=sass` option, this will be a `.sass`
 file instead, and the same applies to `scss/less/styl`.
 
-You can add more global styles via the `apps[0].styles` property in `.angular-cli.json`.
+You can add more global styles via the `styles` option inside your project's build target options
+in `angular.json`.
 These will be loaded exactly as if you had added them in a `<link>` tag inside `index.html`.
 
 ```json
-"styles": [
-  "styles.css",
-  "more-styles.css",
-],
+"architect": {
+  "build": {
+    "builder": "@angular-devkit/build-angular:browser",
+    "options": {
+      "styles": [
+        "src/styles.css",
+        "src/more-styles.css",
+      ],
```

Comment:
please close curly braces

Reply:
Added `...` to indicate more options are there.